yarn-project/acir-simulator/src/public/index.test.ts

```diff
@@ -308,11 +311,8 @@ describe('ACIR public execution simulator', () => {
       // Assert the commitment was created
       expect(result.newCommitments.length).toEqual(1);
 
-      const expectedNoteHash = pedersenPlookupCommitInputs(
-        wasm,
-        params.map(a => a.toBuffer()),
-      );
-      const storageSlot = new Fr(2); // matches storage.nr
+      const expectedNoteHash = pedersenPlookupCommitInputs(wasm, [amount.toBuffer(), secretHash.toBuffer()]);
```

Comment:
oo good catch - lemme figure that out

Reply:
Just to give an update on this. We decided that we will keep on using direct pedersen calls for non-production code because it felt excessive to be implementing individual cbinds just for tests.